lib/Send/CopySentMessageHandler.php

```diff
@@ -88,4 +105,64 @@ public function process(Account $account, LocalMessage $localMessage): LocalMess
 
 		return $this->processNext($account, $localMessage);
 	}
+
+	private function buildRaw(Account $account, LocalMessage $localMessage): \Horde_Mime_Mail {
```

Comment:
would it be possible to deduplicate code from ``\OCA\Mail\Service\MailTransmission::sendMessage``?

Reply:
We could move it to the TransmissionService. Shall I do the refactoring in this PR?

Reply:
Will it make the changes smaller if we have less duplicated code?

Reply:
leaving this in for now - I've tried unsuccessfully to keep the whole code in tune with the $localMessage having to be returned to the chain when there's an SMIME error. I'm probably just too f*king stupid to make it work, so sorry in advance.